### DISCUSSION TITLE
No backpack for defaults.

### DISCUSSION THREAD

**Discussion by toetagger** [GeorgH93/Minepacks#292] [Ideas]
Hi, I cant believe im the first to suggest this but I couldnt find anything when I searched. 

But... make it so when permission for backpack.user is set to false. It doesnt give them the physical backpack. Especially when its locked to their inventory it will just mean that its taking up a valuable slot they dont have use of. 

Either that or have a seperate permission for it?

Thanks
Andy / Cpt.FishPants.

**Comment by GeorgH93** [comment]
They should not receive the item if `backpack.user` is set to `false`, since it will also set `backpack.use` to `false` which is the permission controlling the item.
However the item will not be removed if the user does no longer have the permission, their inventory will have to be cleaned in order for the item to be removed. I guess I should add an additional check to auto remove it. :D
